libyul/optimiser/Suite.h

```diff
@@ -32,6 +32,13 @@
 #include <string_view>
 #include <memory>
 
+#undef OUTPUT_PERFORMANCE_METRICS
+//#define OUTPUT_PERFORMANCE_METRICS 1
```

Comment:
I think it should have a more specific name unless it's intended to be reused for other components too. How about `PROFILE_OPTIMIZER_SEQUENCE`?